deploy/scripts/src/accounts/test.ts

```diff
@@ -216,459 +267,365 @@ const phoneData = {
 }
 
 export function changeEmail (): void {
+  const groups = groupMap.changeEmail
   let res: Response
   const timestamp = new Date().toISOString().slice(2, 16).replace(/[-:]/g, '') // YYMMDDTHHmm
   const iteration = exec.scenario.iterationInInstance.toString().padStart(6, '0')
   const newEmail = `perftest${timestamp}${iteration}@digital.cabinet-office.gov.uk`
   iterationsStarted.add(1)
 
-  res = group('B01_ChangeEmail_01_LaunchAccountsHome GET', () =>
-    timeRequest(() => http.get(env.envURL, {
-      tags: { name: 'B01_ChangeEmail_01_LaunchAccountsHome' }
-    }),
-    { isStatusCode200, ...pageContentCheck('Services you can use with GOV.UK One Login') }))
+  res = group(groups[0], () => timeRequest(() => // B01_ChangeEmail_01_LaunchAccountsHome
+    http.get(env.envURL), { isStatusCode200, ...pageContentCheck('Services you can use with GOV.UK One Login') }))
```

Comment:
```suggestion
    http.get(env.envURL),
    { isStatusCode200, ...pageContentCheck('Services you can use with GOV.UK One Login') }
  ))
```
Suggestion: Don't inline all the calls on this one line

Reply:
Updated